src/hooks/useRecordTypeLabel/useRecordTypeLabel.tsx

```diff
@@ -0,0 +1,8 @@
+import {useRecordTypes} from '../useRecordTypes';
+
+
+export const useRecordTypeLabel = (targetID: string = '') => {
+  const { recordTypes = [] } = useRecordTypes();
+
+  return recordTypes.find(({id}) => id === targetID)?.label || '';
```

Comment:
Nit: use `??`, it's safer
```suggestion
  return recordTypes.find(({id}) => id === targetID)?.label ?? '';
```